xcp_d/utils/restingstate.py

```diff
@@ -51,7 +51,7 @@ def compute_2d_reho(datat, adjacency_matrix):
         rankmean = np.sum(rankeddata, axis=0)  # add up ranks
         # kc is the sum of the squared rankmean minus the timepoints into
         # the mean of the rankmean squared
-        kc = np.sum(np.power(rankmean, 2)) - n_volumes * np.power(np.mean(rankmean), 2)
+        kc = np.sum(np.power(rankmean, 2)) - n_volumes * np.power(np.nanmean(rankmean), 2)
```

Comment:
Can n_volumes be out of sync with rankmean if there are a variable number of nans?

Reply:
That's a good point. I can drop the nanmean in thise case, since the number of volumes comes into play.